test/options/integral/detect/test.js

```diff
@@ -9,6 +9,8 @@ describe('Option `integral`, detect', function() {
       let expected = JSON.parse(JSON.stringify(test.Comb.getConfig('csscomb')));
       delete expected['sort-order'];
       delete expected.exclude;
+      // The default csscomb.json has no tab-size option.
+      expected['tab-size'] = 4;
```

Comment:
The tab-size = 4 is pretty uncommon set up (like in the integral.expected.css file). Putting this into default csscomb.json we'll force users to use this tab-size or overwrite it with their local .csscomb.json. Furthermore, this may introduce test fail on existing or future *.[css, sass, scss, less] test files as most of them have  ruleset blocks indented with 2 white spaces (which will be detected as tab-size = 2).